app/services/change_case_coronavirus_status.rb

```diff
@@ -0,0 +1,68 @@
+class ChangeCaseCoronavirusStatus
```

Comment:
Suggest renaming this to be consistent with the other update services and the wording in the UI (the button). Although we do have some other services prefixed with `Change` so maybe it doesn’t matter.

```suggestion
class UpdateCaseCoronavirusStatus
```

<img width="308" alt="Screenshot 2020-10-28 at 15 39 48" src="https://user-images.githubusercontent.com/30665/97459575-d3100100-1933-11eb-848e-359085eda42d.png">

Reply:
We have as many services using the `Change` prefix currently. Agree we should standardise the terminology (as we should with investigation/case), but that can be done separately.